### PR TITLE
adding a few reminders to the project

### PR DESCRIPTION
**Pull request by jarrodmjack**
## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by jarrodmjack** [comment]
Adding some extra items to the reminders text

File: reminders.txt [hunk 1]
```diff
@@ -45,6 +46,7 @@ Social Reminders:
 - Hey human, remember to have some family time (assuming you have a supportive family, of course)
 - Hey human, if you feel stressed, it's ok to reach out to your loved ones :)
 - Hey human, if you are feeling down, maybe connect with a friend?
+- Hey human, give your family a hug.
```
Comment (ykdojo) [inline reminders.txt]: can you say something like, "assuming you have a supportive family"?

Comment (jarrodmjack) [reply reminders.txt]: I will adjust it and resubmit

Comment (jarrodmjack) [reply reminders.txt]: changed to "loved ones" is this okay?

Comment (ykdojo) [reply reminders.txt]: sounds good

**Review by ykdojo — APPROVED** [review]

**Pull request merged**